src/.flake8

```diff
@@ -13,5 +13,5 @@ exclude=.git,
         tests,
         scripts,
         */settings_commons.py
-max-complexity=6
+max-complexity=7
```

Comment:
This is not "allowed", well we do never do this.

Reply:
the complexity is in _flatten_edge which is a copy from GOB-Export which has no complexity check. then we have to refactor _flatten_edge (which I tried to avoid) or not to use it.